movie-mlops-project/preprocessing/preprocessing.py

```
@@ -0,0 +1,421 @@
+import pandas as pd
+import numpy as np
+import json
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)